release-notes/2023058-ovn-gw-node.md

```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
+Fix issue with spawning gateway nodes in cloud prepare for OVN-Kubernetes CNI in Openstack
```

Comment:
I'll let @dfarrell07 @tpantelis to comment on this.

Personally, my suggestion would be ....
```
Fix issues while spawning Gateway nodes during cloud prepare for clusters deployed on OpenStack environment running OVN-Kubernetes CNI.
```

Reply:
Done.

Reply:
That's fine except I still don't think present tense makes sense, ie "Fixed issues ....".  "Fix issues" sounds like it's still pending or an instruction to do so. But that's my opinion...